java/client/src/main/java/glide/api/models/BaseTransaction.java

```diff
@@ -114,6 +115,20 @@ public T info(InfoOptions options) {
         return getThis();
     }
 
+    /**
+     * Removes the specified <code>keys</code>. A key is ignored if it does not exist.
+     *
+     * @see <a href="https://redis.io/commands/del/">redis.io</a> for details.
+     * @param keys the keys we wanted to remove.
+     * @return the number of keys that were removed.
```

Comment:
Good point.  We should use existing javadoc tags so that the documentation is properly formatted.  Of course, this is different in each language. 

With `@returns` (IntelliJ - Java docs aren't generated by default in VSCode): 
<img width="533" alt="image" src="https://github.com/aws/glide-for-redis/assets/104530826/07627829-4f60-4b5b-957a-1d24af0d844f">

But if I use `Commands Response:`, the javadoc will get all merged together unless I add a bunch of html tags (this is with IntelliJ): 
<img width="523" alt="image" src="https://github.com/aws/glide-for-redis/assets/104530826/94ebc579-8963-43c0-8a57-aa5d3eb06604">

And the python javadoc with `Commands Response:` formats like so (VSCode): 
<img width="554" alt="image" src="https://github.com/aws/glide-for-redis/assets/104530826/e28e3e4b-d73d-4239-9e19-0c57e200cf5a">

Reply:
IMO adding a custom tag is a great idea :)

Reply:
@shachlanAmazon what do you think?